src/test/java/org/mandas/docker/client/DefaultLogStreamTest.java

```diff
@@ -45,12 +45,13 @@ public void testAttach() throws Exception {
         null
     );
 
-    final ByteArrayOutputStream stdout = new ByteArrayOutputStream();
-    final ByteArrayOutputStream stderr = new ByteArrayOutputStream();
-    logStream.attach(stdout, stderr);
+    try (final ByteArrayOutputStream stdout = new ByteArrayOutputStream();
+    		final ByteArrayOutputStream stderr = new ByteArrayOutputStream()) {
+    	logStream.attach(stdout, stderr);
```

Comment:
nit: extra indent

Why did you need to add the try-with-resources here?

Reply:
Didn't resist the temptation to close the byte array streams (resource leaks even in tests can cause problems as well)